src/Sulu/Bundle/AdminBundle/Resources/js/components/ButtonGroup/README.md

```diff
@@ -0,0 +1,62 @@
+The `ButtonGroup` component displays a group of buttons.
+
+Example with two buttons:
+
+```javascript
+const onClick = () => {
+    /* do click things */
+    alert('Clicked this nice button, congrats!');
+};
+
+<ButtonGroup>
+    <Button
```

Comment:
I think I would go for single line here, because there is enough space. Please also check all other appearances in this file. And if we break the lines the `>` should go into its own line.